src/PageSizeSpinBox.h

```diff
@@ -0,0 +1,17 @@
+#ifndef POWEROFTWOSPINBOX_H
+#define POWEROFTWOSPINBOX_H
+
+#include <QSpinBox>
+
+class PageSizeSpinBox : public QSpinBox
+{
+    Q_OBJECT
+public:
+    PageSizeSpinBox(QWidget* parent = Q_NULLPTR);
```

Comment:
I'd prefer a ```nullptr``` here instead of the Qt specific macro.

Reply:
I think it added it like this by default, but I'm not sure.

Reply:
Yeah, it does that. I think the default code snippets that are generated are made to work on all platforms and settings. But since our code only build with C++11 anyway we can use ```nullptr``` here.